### PR TITLE
slash commands

### PR DESCRIPTION
**Pull request by mikael-lagerbom**
This PR aims to migrate almost all of Ripbot's functionality to use Discords slash commands.

We've also upgraded Node to LTS 18, and brought some packages up tod ate.

### CODE REVIEW

File: docker-compose-deploy.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Compose file for local dev, not for prod
```
Comment (TheMeteoRain) [inline docker-compose-deploy.yml:1]: File is named deploy but comment says it's not for prod

File: src/knex.js [hunk 2]
```diff
@@ -1,7 +1,7 @@
 const knex = require('knex');
 
-const { devPool, prodPool } = require('./knexfile');
+const { devPool } = require('./knexfile');
 
-const Knex = knex(process.env.NODE_ENV === 'production' ? prodPool : devPool);
+const Knex = knex(devPool);
```
Comment (TheMeteoRain) [inline src/knex.js:5]: devPool can handle all the environments by changing environment variables? If so fix name. It's not only dev anymore

File: docker-compose-deploy.yml (hunk 1 above)
Comment (mikael-lagerbom) [reply docker-compose-deploy.yml:1]: true

File: src/knex.js (hunk 2 above)
Comment (mikael-lagerbom) [reply src/knex.js:5]: true

**Pull request merged**